test/xml-generator/Class.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SystemClass Name="Test">
+    <SubsystemInclude Path="Subsystem.xml"/>
+</SystemClass>
```

Comment:
No, `SystemClass` does not need to have it's dedicated file, you can replace `<SubsystemInclude Path="Subsystem.xml"/>` by the content of *Subsystem.xml*.